adafruit_stmpe610.py

```diff
@@ -26,12 +26,19 @@
 import time
 from micropython import const
 
+try:
+    from typing import Dict, List, Optional, Tuple
+    from typing_extensions import Literal
+    from microcontroller import Pin
+    from busio import I2C, SPI
+except ImportError:
+    pass
 
 __version__ = "0.0.0+auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_STMPE610.git"
 
 
-def map_range(x, in_min, in_max, out_min, out_max):
+def map_range(x: float, in_min: int, in_max: int, out_min: int, out_max: int) -> float:
```

Comment:
Sorry, didn't realize this the first time, but all of the arguments can conceivably be `float`.